Makefile

```diff
@@ -162,7 +168,16 @@ help:
 	@echo "         make TARGET WORKDIR=/tmp "
 	@echo ""
 	@echo "- REPO_URL"
-	@echo "use it to change the where rootfs base packages will be downloaded"
+	@echo "Use it to change the where rootfs base packages will be downloaded."
+	@echo ""
+	@echo "- IMG_SIZE"
+	@echo "Change the image size of the image to generate."
+	@echo ""
+	@echo "- PKG_MANAGER"
+	@echo "Specify the path to dnf or yum."
+	@echo ""
+	@echo "- USE_DOCKER"
+	@echo "Use a docker container to build an image (useful when depedencies are not installed)."
```

Comment:
Could you add `EXTRA_PKGS` to this list? Also, neither the `README` nor the `Makefile` mention the `DEBUG` variable.

To avoid having to maintain this list of variables in two places, I think it might be worth removing the list of variables in the "Environment Variables" section in the `README` and just saying in that section:

```
Run `make help` to see a list of supported environment variables that can be used to change the tools behaviour.
```